packages/cubejs-cli/src/command/typegen.ts

```diff
@@ -64,13 +64,11 @@ const generateQueryTypes = async (apiUrl, { token }) => {
   };
 
   try {
-    meta = await rp({
-      headers: {
-        authorization: token,
-      },
-      url: `${apiUrl}/meta`,
-      json: true,
-    });
+    const response = await fetch(
+      `${apiUrl}/meta`,
+      { headers: { authorization: token } }
```

Comment:
Maybe its worth adding json header
```suggestion
      { 
        headers: { 
           authorization: token,
           'Content-type': 'application/json',
         }
       }
```